debian/rules

```diff
@@ -1,114 +1,29 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# GNU copyright 1997 to 1999 by Joey Hess.
 
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CFLAGS = -Wall -Wno-format -g
+CFLAGS_MAINT  += -Wall -Wno-format -Wno-format-security -g -DTAP_AND_HOLD
+LDFLAGS_MAINT += -Wl,--as-needed
+
+export DEB_CFLAGS_MAINT_APPEND  = $(CFLAGS_MAINT)
+export DEB_LDFLAGS_MAINT_APPEND = $(LDFLAGS_MAINT)
 
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
-ifeq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
-	INSTALL_PROGRAM += -s
-endif
-ifneq (,$(findstring thumb,$(DEB_BUILD_OPTIONS)))
-        CFLAGS += -mthumb
-endif
 ifeq (,$(findstring nolauncher,$(DEB_BUILD_OPTIONS)))
 	CONFIGURE_OPTIONS += --enable-maemo-launcher
 	USE_MAEMO_LAUNCHER = true
```

Comment:
Where is USE_MAEMO_LAUNCHER used?

Reply:
It is used now to check if we should run dh_maemolauncher.